VideoApp/VideoApp/Stores/Auth/Community/CommunityAuthStore.swift

```diff
@@ -18,28 +18,79 @@ import Foundation
 
 class CommunityAuthStore: AuthStoreEverything {
     weak var delegate: AuthStoreWritingDelegate?
-    var isSignedIn: Bool { return true }
-    var userDisplayName: String { return "Unknown" }
+    var isSignedIn: Bool { keychainStore.passcode != nil }
+    var userDisplayName: String { appSettingsStore.userIdentity }
+    private let appSettingsStore: AppSettingsStoreWriting
+    private let keychainStore: KeychainStoreWriting
+    private let api: APIConfiguring & APIRequesting
+
+    init(
+        appSettingsStore: AppSettingsStoreWriting,
+        keychainStore: KeychainStoreWriting,
+        api: APIConfiguring & APIRequesting
+    ) {
+        self.appSettingsStore = appSettingsStore
+        self.keychainStore = keychainStore
+        self.api = api
+    }
 
     func start() {
+        guard let passcode = keychainStore.passcode else { return }
+        
+        configureAPI(passcode: passcode)
+    }
 
+    func signIn(email: String, password: String, completion: @escaping (AuthError?) -> Void) {
+        
```

Comment:
I pushed changes with a simple log message for now. In the future I would like to create a logging wrapper that I can use to log an error and it will both print message and `fatalError()` when debug build. 

I like the idea of returning an error but to do that right would involve injecting a queue to the object because the function really should be async so probably makes sense to just log error.